### DISCUSSION TITLE
NZB Renaming

### DISCUSSION THREAD

**Discussion by SteffanCline** [nzbgetcom/nzbget#373] [General]
Is there an email address where we can send nzbs that don't rename properly in NZBGet but do in SabNZB? 

I have one that downloaded in NZBGet and still had the names all randomized so I retried it in SabNZBD and it worked fine.

**Comment by dnzbk** [comment]
Just send it to me (denis@nzbget.com) and I'll test it.

**Reply by SteffanCline** [reply]
Sent directly to you via your email address above.

**Comment by dnzbk** [comment]
Confirmed and transformed the discussion into the [issue](https://github.com/nzbgetcom/nzbget/issues/380).
Thank you for the report.
